app/controllers/posts_controller.rb

```diff
@@ -1,10 +1,12 @@
 class PostsController < ApplicationController
   def index
-    @user_index = params[:user_id]
+    @user_path = "/users/#{params[:user_id]}"
+    @user = User.find(params[:user_id])
+    @posts = Post.where(author_id: params[:user_id])
```

Comment:
- [ ] <b>REQUIRED:</b> There is no need of doing another query here, you can get the post that belongs to that user by simply `@posts = @user.posts`.